package.yaml

```diff
@@ -1,21 +1,23 @@
-name:       conversions
-synopsis:   Injective explicit total and partial conversions
-homepage:   https://github.com/mbj/conversions#readme
-maintainer: Markus Schirp mbj@schirp-ds.com, Allan Lukwago <epicallan.al@gmail.com>
 copyright:  2020 Strainge Inc.
 github:     mbj/conversions
+homepage:   https://github.com/mbj/conversions#readme
+license:    MIT
+maintainer: Markus Schirp mbj@schirp-ds.com, Allan Lukwago <epicallan.al@gmail.com>
+name:       conversions
+synopsis:   Injective explicit total and partial conversions
 version:    0.0.2
 
 dependencies:
-- base
-- bytestring
-- control-bool
-- devtools
-- exceptions
-- mtl
-- source-constraints
-- template-haskell
-- text
+- base               > 4.12 && < 14.15
```

Comment:
The `14.15` on this line should probably be `4.15`

Reply:
Unless we had a build on `8.6` that works I'd not want to declare 8.6 support. `source-constraints` etc do not support 8.6 at this point. 

I'd rather spend my energy forwards than backwards.

Reply:
> Unless we had a build on 8.6 that works I'd not want to declare 8.6 support. source-constraints etc do not support 8.6 at this point.
I'd rather spend my energy forwards than backwards.

Makes sense.